pkgs/phps.nix

```diff
@@ -85,6 +85,8 @@ let
               done
             ''
             + attrs.preConfigure;
+
+          meta = builtins.removeAttrs attrs.meta [ "knownVulnerabilities" ];
```

Comment:
I think this would just remove it from `php`.

Reply:
So, I remove it from `php` and all the extensions I can find then ?

Reply:
The package that has it set is `openssl`, not `php`.

Reply:
Right, then what I'm doing here is totally wrong... and I have no clue how to only alter the `openssl` dependency.

Reply:
Again, it is not attribute on extension. It is on `pkgs.openssl`.

Reply:
You can do it in an overlay here:
https://github.com/fossar/nix-phps/blob/3f15ee063c4a83185205ea7fd69ad025fbf7c283/flake.nix#L27

Reply:
Ok it's working! I've learned something, thanks!